tests/object_storage_deleter_test.php

```diff
@@ -0,0 +1,132 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+namespace tool_objectfs\tests;
+
+defined('MOODLE_INTERNAL') || die();
+
+use tool_objectfs\object_file_system;
+use tool_objectfs\object_manipulator\object_storage_deleter;
+
+require_once(__DIR__ . '/classes/test_client.php');
+require_once(__DIR__ . '/tool_objectfs_testcase.php');
+
+class object_storage_deleter_test extends tool_objectfs_testcase {
```

Comment:
This test doesnt need a manipulator - just test the test_file_system directly inside object_file_system_testcase